.github/test/test-workflows.js

```
@@ -0,0 +1,193 @@
+      'github-stats.yml': '0 */4 * * *',
+      'my-badges.yml': '0 0 * * *',
+      'update-readme.yml': '10 */4 * * *'
+    };
+    
+    Object.entries(expectedSchedules).forEach(([file, expectedCron]) => {
+      const filePath = path.join(WORKFLOWS_DIR, file);
+      if (fs.existsSync(filePath)) {
+        const content = fs.readFileSync(filePath, 'utf8');
+        const workflow = yaml.load(content);
+        
+        if (workflow.on && workflow.on.schedule) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)